components/camel-rabbitmq/src/main/java/org/apache/camel/component/rabbitmq/RabbitMQDeclareSupport.java

```diff
@@ -150,6 +155,18 @@ private void declareAndBindQueue(
         }
     }
 
+    private void passivelyDeclareExchangeAndQueueAndBindThem(
+            final Channel channel, final String queue, final String exchange, final String routingKey,
+            final Map<String, Object> bindingArgs)
+
+            throws IOException {
+
+        //if (!endpoint.isPassive()) { log.warn("Forcing passive because declaration was skipped"); }
```

Comment:
should this code comment be removed?

Reply:
I didn't know what to do with a situation where the user may have defined passive=false but left the skip-declarations enabled.
I was thinking that logging (warning) this detail would be a good idea but I don't know if there is a logging policy (and the class does not even have its own logger).
I was hoping someone could enlighten me. I can clean it up either way